resources/app/index.js

```diff
@@ -1,10 +1,137 @@
-let mappedSlash
-navigator.keyboard.getLayoutMap().then((keyboardLayout) => {
-  mappedSlash = keyboardLayout.get('Slash')
-})
-
-export const getSummonShortcut = (platform) => {
-  const modifierKey = platform === 'darwin' ? 'Option' : 'Alt'
-  const summonKey = mappedSlash ? mappedSlash : '/'
-  return { modifierKey, summonKey }
+const shortcutKeyMap = {
```

Comment:
can we make this file TS

Reply:
I'll give it a go

Reply:
@mholtzman I think this is part of a larger piece of work around enabling TS in renderer files

Reply:
were you running into some issues here? I simply changed this file to `index.ts` and it works fine